operator/cmd/hubble_linux.go

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/cilium/cilium/pkg/hive"
+	"github.com/cilium/cilium/pkg/option"
+	"github.com/microsoft/retina/operator/cmd/hubble"
+	"github.com/spf13/cobra"
+)
+
+var (
+	h         = hive.New(hubble.Operator)
+	hubbleCmd = &cobra.Command{
```

Comment:
This is not `HubbleCmd`, please rename variable.

Reply:
renamed to cmd.